src/modules/learning-object-service/collections/CollectionController.ts

```diff
@@ -126,6 +126,53 @@ export class CollectionController implements Controller {
      */
     router.patch('/learning-objects/:learningObjectId/collections', this.proxyRequest((req: Request) => LEARNING_OBJECT_ROUTES.ADD_LEARNING_OBJECT_TO_COLLECTION(req.params.learningObjectId)));
 
+    /**
+     * @swagger
+     * /users/{username}/learning-objects/{cuid}/collection:
+     *  patch:
+     *    description: Updates a in review object's collection
+     *    tags:
+     *      - Learning Object Service
+     *    parameters:
+     *      - in: path
+     *        name: username
+     *        schema:
+     *            type: string
+     *        required: true
+     *        description: The username of the author
+     *      - in: path
+     *        name: cuid
+     *        schema:
+     *            type: string
+     *        required: true
+     *        description: The cuid of the object
+     *    requestBody:
+     *      description: The collection
+     *      required: true
+     *      content:
+     *        application/json:
+     *          schema:
+     *            type: object
+     *            properties:
+     *              collection:
+     *                type: string
+     *                example: nccp
+     *                description: The collection name
+     *                required: true
+     *    responses:
+     *      200:
+     *        description: OK
+     *      400:
```

Comment:
Another reason for a 400 could be an invalid collection name passed up.

Reply:
What would qualify a invalid collection name?

Reply:
Any collection that is not in the collections mongo collection. For the featured stuff I did last week I pulled all the collections and verified that the learning object had a collection that actually exists and the collection that was going to be getting objects actually existed in our system.

Reply:
Wouldn't that be a 404?  Because it can't find the collection specified

Reply:
That's fine as long as it's handled and we're not moving objects to collections that don't exist.